zuul/connection/github.py

```diff
@@ -165,6 +165,26 @@ def _event_issue_comment(self, request):
         event.type = 'pr-comment'
         return event
 
+    def _event_pull_request_review(self, request):
+        """Handles pull request reviews"""
+        body = request.json_body
+        action = body.get('action')
+        if action != 'submitted':
+            return
+        pr_body = body.get('pull_request')
+        if pr_body is None:
+            return
+
+        review = body.get('review')
+        if review is None:
+            return
+
```

Comment:
This makes sense to me... we're making sure that the body object has a review key before transforming the entire body object into an event.